CONTRIBUTING.md

```diff
@@ -66,6 +66,17 @@ If you would like to add new capabilities to the product as described in **B)**,
 
     Once the issue is created on GitHub, it still needs to be approved. Once approved, the rest of the process is identical to the process described in **A)**.
 
+## Get to Know the System App
+
+Before updating the System Application, familiarize yourself with the following sites on Microsoft Learn:
+
+- [Module architecture](https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-blueprint)
+- [Get started with modules](https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-blueprint)
+- [Set up your development environment](https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-set-up-an-environment)
+- [Create a new module](https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-new-module)
+- [Create a .NET wrapper module](https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-create-a-wrapper-module)
+- [Change a module](https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-change-a-module)
```

Comment:
```suggestion
- [Module architecture](https://learn.microsoft.com/dynamics365/business-central/dev-itpro/developer/devenv-blueprint)
- [Get started with modules](https://learn.microsoft.com/dynamics365/business-central/dev-itpro/developer/devenv-blueprint)
- [Set up your development environment](https://learn.microsoft.com/dynamics365/business-central/dev-itpro/developer/devenv-set-up-an-environment)
- [Create a new module](https://learn.microsoft.com/dynamics365/business-central/dev-itpro/developer/devenv-new-module)
- [Create a .NET wrapper module](https://learn.microsoft.com/dynamics365/business-central/dev-itpro/developer/devenv-create-a-wrapper-module)
- [Change a module](https://learn.microsoft.com/dynamics365/business-central/dev-itpro/developer/devenv-change-a-module)
```

Reply:
Wow, didn't know it was possible to remove the language in the url. Excellent suggestion